includes/Withdraw/Hooks.php

```diff
@@ -87,4 +103,109 @@ public static function update_vendor_balance( $withdraw ) {
 
         self::delete_seller_balance_cache( $withdraw->get_status(), $withdraw->get_user_id(), $withdraw->get_id() );
     }
+
+    /**
+     * Handle withdraw request ajax.
+     *
+     * @since 3.3.1
+     *
+     * @return void
+     */
+    public function handle_withdraw_request() {
+        if ( ! isset( $_POST['nonce'] ) || ! wp_verify_nonce( sanitize_key( $_POST['nonce'] ), 'dokan_withdraw' ) ) {
+            wp_send_json_error( esc_html__( 'Are you cheating?', 'dokan-lite' ) );
+        }
+
+        $user_id = dokan_get_current_user_id();
+
+        if ( dokan()->withdraw->has_pending_request( $user_id ) ) {
+            wp_send_json_error( esc_html__( 'You already have a pending withdraw request.', 'dokan-lite' ) );
+        }
+
+        if ( ! current_user_can( 'dokan_manage_withdraw' ) ) {
+            wp_send_json_error( esc_html__( 'You have no permission to do this action', 'dokan-lite' ) );
+        }
+
+        if ( ! isset( $_POST['method'] ) ) {
+            wp_send_json_error( esc_html__( 'Withdraw method is required', 'dokan-lite' ) );
+        }
+
+        if ( empty( $_POST['amount'] ) ) {
+            wp_send_json_error( esc_html__( 'Withdraw amount is required', 'dokan-lite' ) );
+        }
+
+        $amount = (float) wc_format_decimal( wp_unslash( $_POST['amount'] ) );
+        $method = wc_clean( wp_unslash( $_POST['method'] ) );
```

Comment:
if `$_POST['method']` is expected to be a string and not an array, you can simply sanitize it using `sanitize_text_field`